apps/pip/account-management/ithc.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: pip-account-management
+  namespace: pip
+spec:
+  releaseName: pip-account-management
+  values:
+    image: sdshmctspublic.azurecr.io/pip/account-management:prod-6f844de-20220105164733 #{"$imagepolicy": "flux-system:pip-account-management"}
+    environment: "ithc"
+    resourceGroup: "pip-ss-ithc-rg"
+    subscriptionId: "ba71a911-e0d6-4776-a1a6-079af1df7139"
+    tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+    managedIdentityClientId: "6eadcd84-611f-4d93-b2c4-5e3ff2dc239e"
```

Comment:
this doesn't look right, I don't think any of these values should be here

Reply:
try without? those should be added by an environment level helm release

Reply:
Left the environment in since the chart values require global.environment and not sure if it's set anywhere else. Will need to add ithc values file to the chart repos for the apps